src/Tribe/Date_Utils.php

```diff
@@ -351,5 +306,223 @@ public static function week_ends_on( $week_starts_on ) {
 		}
+
+		// DEPRECATED METHODS
+		// @codingStandardsIgnoreStart
+		/**
+		 * Deprecated camelCase version of self::date_only
+		 *
+		 * @param int|string $date        The date (timestamp or string).
+		 * @param bool       $isTimestamp Is $date in timestamp format?
+		 *
+		 * @return string The date only in DB format.
+		 */
+		public static function dateOnly( $date, $isTimestamp = false ) {
+			_deprecated_function( __FILE__, '3.11', __CLASS__ . '::date_only' );
+			return self::date_only( $date, $isTimestamp );
+		}
+
+		/**
+		 * Deprecated camelCase version of self::time_only
+		 *
+		 * @param string $date The date.
+		 *
+		 * @return string The time only in DB format.
+		 */
+		public static function timeOnly( $date ) {
+			_deprecated_function( __FILE__, '3.11', __CLASS__ . '::time_only' );
+			return self::time_only( $date );
+		}
+
+		/**
+		 * Deprecated camelCase version of self::hour_only
+		 *
+		 * @param string $date The date.
+		 *
+		 * @return string The hour only.
+		 */
+		public static function hourOnly( $date ) {
+			_deprecated_function( __FILE__, '3.11', __CLASS__ . '::hour_only' );
+			return self::hour_only( $date );
+		}
+
+		/**
+		 * Deprecated camelCase version of self::minutes_only
+		 *
+		 * @param string $date The date.
+		 *
+		 * @return string The minute only.
+		 */
+		public static function minutesOnly( $date ) {
+			_deprecated_function( __FILE__, '3.11', __CLASS__ . '::minutes_only' );
+			return self::minutes_only( $date );
+		}
+
+		/**
+		 * Deprecated camelCase version of self::meridian_only
+		 *
+		 * @param string $date The date.
+		 *
+		 * @return string The meridian only in DB format.
+		 */
+		public static function meridianOnly( $date ) {
+			_deprecated_function( __FILE__, '3.11', __CLASS__ . '::meridian_only' );
+			return self::meridian_only( $date );
+		}
+
+		/**
+		 * Returns the end of a given day.
+		 *
+		 * @deprecated since 3.10 - use tribe_event_end_of_day()
+		 * @todo       remove in 4.1
+		 *
+		 * @param int|string $date        The date (timestamp or string).
+		 * @param bool       $isTimestamp Is $date in timestamp format?
+		 *
+		 * @return string The date and time of the end of a given day
+		 */
+		public static function endOfDay( $date, $isTimestamp = false ) {
+			_deprecated_function( __FILE__, '3.10', 'tribe_event_end_of_day' );
+
+			$date = $isTimestamp ? $date : strtotime( $date );
+			$date = date( self::DBDATEFORMAT, $date );
+			$date = strtotime( $date . ' 23:59:59' );
+
+			return date( self::DBDATETIMEFORMAT, $date );
+		}
+
+		/**
+		 * Returns the beginning of a given day.
+		 *
+		 * @deprecated since 3.10
+		 * @todo       remove in 4.1
+		 *
+		 * @param int|string $date        The date (timestamp or string).
+		 * @param bool       $isTimestamp Is $date in timestamp format?
+		 *
+		 * @return string The date and time of the beginning of a given day.
+		 */
+		public static function beginningOfDay( $date, $isTimestamp = false ) {
```

Comment:
I know you just relocated these deprecated functions (`endOfDay` as well), but why do they have repeated functionality rather than calling the new functions?